_posts/2018/2018-04-29-use-hash-fetch-when-using-params-in-controllers.md

```diff
@@ -0,0 +1,77 @@
+---
+title: 'Use Hash#fetch when using Rails params in controllers'
+description: ''
+layout: article
+category: ruby
+image:
+  base: '2018/use-hash-fetch-when-using-params-in-controllers'
+  alt: 'Retriever'
+  source: 'https://unsplash.com/photos/Pzu9f6Nby5w'
+  credit: 'Mitchell Orr'
+---
+
+We often need to access a parameter from the URL that isn't part of the regular rails routes. Perhaps a query parameter such as `/search?q=term`.
```

Comment:
*Rails

Reply:
For the second sentence: For example, _the_ query parameter in '/search?q=term'.